mode/local_connection_extractor.go

```diff
@@ -104,6 +110,9 @@ func parseConnectionInfo(s *terraform.InstanceState) (*connectionInfo, error) {
 		}
 		if connInfo.BastionPrivateKey == "" {
 			connInfo.BastionPrivateKey = connInfo.PrivateKey
+		} else {
+			block, _ := pem.Decode([]byte(connInfo.BastionPrivateKey))
+			connInfo.BastionPrivateKey = string(pem.EncodeToMemory(block))
```

Comment:
Is it possible to check here what has been decoded? The example here: https://golang.org/pkg/encoding/pem/#example_Decode suggests so. Would that be a good idea?

Reply:
Good idea how does this look?